Watchman.Discord/Areas/Messaging/Services/SendRequestParser.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Linq;
+using Devscord.DiscordFramework.Commons.Exceptions;
+using Devscord.DiscordFramework.Framework.Commands.Parsing.Models;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+using Devscord.DiscordFramework.Services;
+using Watchman.Discord.Areas.Commons;
+
+namespace Watchman.Discord.Areas.Messaging.Services
+{
+    public class SendRequestParser
+    {
+        private DiscordRequest _request;
+        private Contexts _contexts;
+        private ChannelsService _channelsService;
+
+        public SendRequestParser(DiscordRequest request, Contexts contexts, ChannelsService channelsService) {
```

Comment:
musisz tworzyć wiele instancji tego obiektu -> bez sensu obciążasz procesor i pamięć
jeśli jest taka możliwość to lepiej tego unikać

z jednej strony to walka o drobne cykle procesora które dużo nie zmieniają
ale z drugiej strony to budowanie dobrych nawyków i oszczędzanie praktycznie zerowym kosztem